src/main/java/net/pms/io/BasicSystemUtils.java

```diff
@@ -233,6 +234,27 @@ public int getPingPacketFragments(int packetSize) {
 		return ((packetSize + 8) / 1500) + 1;
 	}
 
+	/**
+	 * @return whether the computer is running in dark mode
+	 */
+	private boolean isDarkMode() {
+		if (!Platform.isMac()) {
+			return false;
+		}
+
+		try {
+			// check for exit status only. Once there are more modes than "dark" and "default", we might need to analyze string contents..
+			final Process proc = Runtime.getRuntime().exec(new String[]{"defaults", "read", "-g", "AppleInterfaceStyle"});
```

Comment:
Personal nitpick:
```suggestion
                         String cmd = "defaults read -g AppleInterfaceStyle";
			final Process proc = Runtime.getRuntime().exec(cmd);
```

Reply:
@js-kyle The only problem is that the result isn't the same :wink: The working code places each argument as an element in a string array, your suggestion creates one space-separated string.